validator/engine/validator.js

```diff
@@ -2867,6 +2875,22 @@ class Context {
   getEncounteredBodyLineCol() {
     return /** @type {!LineCol} */ (this.encounteredBodyLineCol_);
   }
+
+  /**
+   * Record if this document contains a tag requesting the LTS runtime engine.
+   * @param {!ParsedTagSpec}
+   * @private
+   */
+  recordLtsEngineUsedFromTagSpec_(parsedTagSpec) {
+    if (parsedTagSpec.getSpec().specName === 'amphtml engine v0.js script [AMP LTS]') {
```

Comment:
@sedano I wasn't certain if this was the right way of checking which spec was validated. Should I be using an ID, or a constant somewhere?